src/richie/apps/courses/templates/courses/cms/course_detail.html

```diff
@@ -106,7 +106,22 @@ <h1 class="subheader__title">
 
                     <div class="subheader__teaser">
                         {% placeholder "course_teaser" or %}
-                        <div class="empty">{% trans 'Add a video or teaser.' %}</div>
+                            {% get_placeholder_plugins "course_cover" as plugins or %}
+                                <p class="empty">{% trans 'Add a video on teaser or add an image for course cover on its glimpse.' %}</p>
```

Comment:
```suggestion
                                <p class="empty">{% trans 'Add a teaser video or add a cover image below and it will be used as teaser image as well.' %}</p>
```